templates/CSI/rbd/secret.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: csi-rbd-secret
-  namespace: default
+  namespace: openshift-storage
```

Comment:
@ebenahar in the secret.yaml files present in rook repo, the secrets are created in **default** namespace. It wouldn't harm to change the namespace to "openshift-storage". 
But maybe we can confirm with dev, if there's any particular reason for creating secrets in "default" project, rather than in "rook-ceph" projects

Reply:
@nehaberry Good point. I was wondering about it. Can you please raise this question to them?

Reply:
@ebenahar sure will ask Madhu and update here.